.travis.yml

```diff
@@ -14,8 +14,8 @@ script:
 notifications:
   irc:
     channels:
-      - "chat.freenode.net#wikidata-feed"
-    on_success: never
```

Comment:
Is this intentional?

Reply:
Lucas "specified" (or at least suggested) it in [the phab ticket description](https://phabricator.wikimedia.org/T283707): "(probably change on_success: never to on_success: change, like in other channels, while we’re at it?)"

Reply:
Noted. Thanks.